public/components/overview/home.tsx

```diff
@@ -67,13 +67,17 @@ export const Home = () => {
             order: card.order,
             description: card.description,
             title: card.title,
-            onClick: () => coreRefs.application?.navigateToApp(card.url, { path: '#/' }),
-            getFooter: () => {
-              return (
-                <EuiText size="s" textAlign="center">
-                  {card.footer}
-                </EuiText>
-              );
+            cardProps: {
+              selectable: {
+                children: (
+                  <EuiI18n
```

Comment:
i don't think we use the one from Eui directly. you can use `FormattedMessage` https://github.com/opensearch-project/OpenSearch-Dashboards/blob/b83fdecd9370e5c73d2f7408084f02b6fc5cae56/packages/osd-i18n/README.md?plain=1#L238-L245

Reply:
Updated to
```
                  <FormattedMessage
                    id="home.sampleData.card.footer"
                    defaultMessage={card.footer || 'Documentation'}
                  />
```